src/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "svg-icons-builder",
+  "version": "1.0.0",
+  "description": "",
+  "private": true,
+  "main": "index.js",
+  "builders": "./builders.json",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "MIT"
+}
```

Comment:
This file is a dummy `package.json` for testing, so marked private and not using the fully qualified name. This is because I don't know another way to wire up the correct pathing in tests and non tests.

Reply:
Okay, I see. Maybe the following could help to get the correct `package.json` inside the `index.spec.ts`.

````
   await architectHost.addBuilderFromPackage(join(workspaceRoot, '..'));
````
In general, I would say that there should be a nicer way of doing this. If the suggested solution doesn't work then maybe we need to refactor the structure according to this [project](https://github.com/mgechev/cli-builders-demo)

Reply:
tried the above solution and no dice. It can find the `package.json` and it can't find `builders.json` because of this line in package.json:
```
"builders": "./builders.json",
```

It's this way because the expectation appears to be that the structure is maintained relative to final output. i.e. if `builders.json` lives in src in the project, there should be a src in `dist`. If keeping `src` dir was desired making `dist` consistent would work.

https://github.com/mgechev/cli-builders-demo has a couple interesting differences. 

1. Being a demo project to execute against, I see this a lot in the wild.
2. No wrapper dir causing the mismatch, I adjusted this PR to account for that.

Reply:
Awesome 👍